### ISSUE TITLE
Returned data for Pekka and Mini Pekka is missing data

### ISSUE THREAD

**Issue by jimmyk3008** [cr-api/cr-api#245]
## Request Details:

```
GET
http://api.cr-api.com/player/8LQUQPRJ
```

## Response Details:
```
Code: 200

+"deck": array:8 [▼
        0 => {#357 ▼
          +"name": "P.E.K.K.A"
          +"level": 4
          +"maxLevel": 8
          +"rarity": "epic"
          +"requiredForUpgrade": 20
          +"icon": "https://api-assets.clashroyale.com/cards/300/MlArURKhn_zWAZY-Xj1qIRKLVKquarG25BXDjUQajNs.png"
        }
        1 => {#356 ▼
          +"name": "Poison"
          +"level": 4
          +"maxLevel": 8
          +"rarity": "Epic"
          +"requiredForUpgrade": 20
          +"icon": "https://api-assets.clashroyale.com/cards/300/98HDkG2189yOULcVG9jz2QbJKtfuhH21DIrIjkOjxI8.png"
          +"key": "poison"
          +"elixir": 4
          +"type": "Spell"
          +"arena": 5
          +"description": "Covers the area in a deadly toxin, damaging enemy troops and buildings over time. Yet somehow leaves the grass green and healthy. Go figure!"
          +"id": 28000009
        }

```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there? Recently
 2. Expected behaviour: return:
* Rarity to have first letter upper case
* Key
* Elixir
* Type
* Arena
* Description
* id

**Comment by fourjr** [comment]
Might be related to #241

**Comment by anksite** [comment]
have a same problem...   need the 'key' of that cards

**Comment by selfish** [comment]
Fixed in `v10.2.19`
